src/components/NotificationsPage.vue

```diff
@@ -49,10 +49,7 @@ export default {
   },
   computed: {
     notifications() {
```

Comment:
This can be removed entirely and replaced with `store.notifications` in the `v-for` directive.